www/htdocs/index.html

```diff
@@ -2619,6 +2626,331 @@
+
+- Works as a daemon
+
+- Offloads collecting, processing, and publishing to plugins
+
+- Have to configure it to use the plugins and collect the metrics you want
+
+---
+
+## InfluxDB
+
+- Since Snap doesn't have have a database, we need one
+
+- It's specifically for time series
+
+---
+
+## Grafana
+
+- Since neither Snap or InfluxDB can show graphs, we're using Grafana
+
+---
+
+## Snap and Global Scheduling
+
+---
+
+## Storage and UI Layers
+
+---
+
+## Getting Snap
+
+Get the latest Snap release
+.exercise[
+
+- Get Snap:
+  ```bash
+    docker run -v /opt:/opt centos sh -c "curl -sSL https://github.com/intelsdi-x/snap/releases/download/v0.15.0-beta/snap-v0.15.0-beta-linux-amd64.tar.gz | tar -C /opt -zxf-"
+  ```
+- Get plugins:
+  ```bash
+    docker run -v /opt:/opt centos sh -c "curl -sSL https://github.com/intelsdi-x/snap/releases/download/v0.15.0-beta/snap-plugins-v0.15.0-beta-linux-amd64.tar.gz | tar -C /opt -zxf-"
+  ```    
+
+]
+
+---
+
+## Setting up your Snap environment
+
+.exercise[
+
+- Export snap version to environment variable
+  ```bash
+    export SNAPVER=v0.15.0-beta
+  ```
+- Create a symlink for Snap
+  ```bash
+  ln -s /opt/snap-$SNAPVER/ /opt/snap
+  ln -s /opt/snap/bin/snapd /usr/local/bin/snapd
+  ln -s /opt/snap/bin/snapctl /usr/local/bin/snapctl
+  ```
+
+  ]
+
+---
+
+## `snapd`- Snap daemon
+
+- Application made up of a REST API, control module, and scheduler module
+
+- Start `snapd` with plugin trust disabled and log level set to debug
+  ```bash
+    snapd -t 0 -l 1
+  ```
+
+]
+
+---
+
+## `snapctl` - loading plugins
+
+.exercise[
+
+- Load the psutil collector plugin
+  ```bash
+    snapctl plugin load plugin/snap-plugin-collector-psutil
+  ```
+
+- Load the file publisher plugin
+  ```bash
+    snapctl plugin load plugin/snap-plugin-publisher-file
+  ```
+
+]
+
+---
+
+## `snapctl` - see what you loaded and can collect
+
+.exercise[
+
+- See your loaded plugins
+  ```bash
+    snapctl plugin list
+  ```
+
+- See the metrics you can collect
+  ```bash
+   snapctl metric list
+  ```
+
+]
+
+---
+
+## Starting a task
+
+.exercise[
+
+- Using the task manifest in the snap directory, start a task to collect metrics from psutil and publish them to a file.
+
+  ```bash
+    snapd task create -t snap/psutil-file.yml
+  ```
+
+  ]
+
+  The output should look like the following:
+  ```
+    Using task manifest to create task
+    Task created
+    ID: 240435e8-a250-4782-80d0-6fff541facba
+    Name: Task-240435e8-a250-4782-80d0-6fff541facba
+    State: Running
+  ```
+
+---
+
+## See the tasks
+
+.exercise[
+
+- Using the task in the snap directory start a task to collect metrics from psutil and publish them to a file.
+
+  ```bash
+    snapctl task list
+  ```
+
+]
+
+The output should look like the following:
+  ```
+    ID           NAME              STATE     HIT MISS FAIL CREATED  LAST FAILURE
+    24043...acba Task-24043...acba Running   4   0    0    2:34PM   8-13-2016 	
+  ```
+---
```

Comment:
I'd suggest to proceed with influxdb and grafana deployment at this point. That way, we can see fancy nifty graphs. And then, move to trickier stuff (deploying snapd all over the place, setting up tribes and whatnots)